### PR TITLE
Use wide unicode build for older Pythons

### PR DESCRIPTION
**Pull request by njsmith**
Closes gh-18

### CODE REVIEW

**Comment by njsmith** [comment]
Maybe needs a test?

File: docker/Dockerfile [hunk 1]
```diff
@@ -45,11 +47,16 @@ RUN export OPENSSL_VERSION=openssl-1.0.2f && \
     (cd ${OPENSSL_VERSION} && ./config no-ssl2 no-shared -fPIC --prefix=/usr/local/ssl && make && make install) && \
     rm -rf ${OPENSSL_VERSION} ${OPENSSL_VERSION}.tar.gz ${OPENSSL_VERSION}.tar.gz.sha256 && \
     for PYVER in ${PY_VERS}; do  \
+        if expr $PYVER : '^2\.' >/dev/null; then \
+            UNICODE_OPT="--enable-unicode=ucs4"  \
+        else \
+            UNICODE_OPT="" \
+        fi;
```
Comment (rmcgibbo) [inline docker/Dockerfile]: missing a `\`

**Comment by rmcgibbo** [comment]
Excellent

**Comment by njsmith** [comment]
This is still broken actually :-) One moment...

**Comment by njsmith** [comment]
Oh thanks goodness. Tests are finally working for real. Ready to merge I think.

**Pull request merged**